naresh/src/main/java/com/lftechnology/phpjava/assignmenttwo/MutiplierChecker.java

```diff
@@ -0,0 +1,56 @@
+package com.lftechnology.phpjava.assignmenttwo;
+
+import com.lftechnology.phpjava.menu.MenuReader;
+import com.lftechnology.phpjava.menu.ProgramMenu;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * Created by naresh on 5/12/16.
+ */
+public class MutiplierChecker implements MenuReader {
+
+    /**
+     * read the user input for the action
+     */
+    @Override
+    public void scanner() {
+        Scanner scan = new Scanner(System.in);
+        System.out.println("Please enter the max value.");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS106)